src/zappy_gui_src/network/api/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ target_include_directories(${TARGET}
 )
 
 set_target_properties(${TARGET} PROPERTIES
-    LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/src/zappy_gui
+    LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/src/zappy_gui_src
 )
```

Comment:
This property should be in the src/zappy_gui_src/CMakelists.txt

Reply:
why ? The target is created in this file, it shouldn't cause any pb right ?

Reply:
The target is created in the file I mentioned, and not in this one